examples/README.md

```diff
@@ -0,0 +1,41 @@
+## Structure
+
+For the simplest examples without using any helping code (see `framework.rs` here), check out:
+  - `hello ` for printing adapter information
+  - `hello-triangle` for graphics and presentation
+  - `hello-compute` for pure computing
+
+Notably, `capture` example show rendering without a surface/window. It reads back the contents and saves them to a file.
+
+All framework-based examples render to the window.
+
+## Feature matrix
+| Feature                | boids  | cube   | mipmap | msaa-line | shadow | skybox | texture-arrays | water  |
+| ---------------------- | ------ | ------ | ------ | --------- | ------ | ------ | -------------- | ------ |
+| vertex attributes      | :star: | :star: | :star: | :star:    | :star: |        | :star:         | :star: |
+| instancing             | :star: |        |        |           |        |        |                |        |
+| lines and points       |        |        |        | :star:    |        |        |                |        |
+| sampled color textures | :star: | :star: | :star: |           |        | :star: | :star:         | :star: |
+| storage textures       | :star: |        |        |           |        |        |                |        |
+| binding array          |        |        |        |           |        |        | :star:         |        |
+| comparison samplers    |        |        |        |           | :star: |        |                |        |
+| subresource views      |        |        | :star: |           | :star: |        |                |        |
+| cubemaps               |        |        |        |           |        | :star: |                |        |
+| multisampling          |        |        |        | :star:    |        |        |                |        |
+| off-screen rendering   |        |        |        |           | :star: |        |                | :star: |
+| stencil testing        |        |        |        |           |        |        |                |        |
+| depth testing          |        |        |        |           | :star: |        |                | :star: |
+| read-only depth        |        |        |        |           |        |        |                | :star: |
+| blending               |        |        |        |           |        |        |                | :star: |
+| render bundles         |        |        |        | :star:    |        |        |                | :star: |
+| compute passes         | :star: |        |        |           |        |        |                |        |
+| optional extensions    |        |        |        |           |        |        |                |        |
```

Comment:
This row should probably also have a star.

Reply:
I was more thinking in line of blanking out the row completely. But a star works too :)